src/objects/avatar/util/getAvatarDrawDefinition.ts

```diff
@@ -238,7 +255,7 @@ function getBodyPart(
     const assets = framesIndexed.map((animationFrame) =>
       getAssetForFrame({
         offsetsData,
-        direction,
+        direction: headComponents.has(part.type as AvatarFigurePartType) && headDirection ? headDirection : direction,
```

Comment:
I'm quite sure this is causing a bug for `headDirection = 0` and will fallback to the avatar direction.
If you have a number type which can also be `undefined`, explicitly check for `undefined` by doing `headDirection != null`.
Otherwise you will get unintended behavior if the value is `0`, since `0` in JavaScript is falsy.

Reply:
Thanks, always forget 0 counts as false too!